golangci-lint breaking with go 1.14

Not sure if this is an issue with podsync or something else but when running `make` in a clean go 1.14 docker image I get this error (with podsync being mounted to `/code` in the container):

```
/code/bin/golangci-lint run
WARN [runner] Can't run linter goanalysis_metalinter: vrp: failed prerequisites: buildssa@github.com/mxpv/podsync/pkg/model 
WARN [runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/code/pkg/model/defaults.go:4:2: could not import time (/usr/local/go/src/time/format.go:7:8: could not import errors (/usr/local/go/src/errors/wrap.go:8:2: could not import internal/reflectlite (/usr/local/go/src/internal/reflectlite/value.go:8:2: could not import runtime (/usr/local/go/src/runtime/alg.go:8:2: could not import internal/cpu (-: could not load export data: cannot import "internal/cpu" (unknown iexport format version 1), export data is newer version - update tool))))) /code/pkg/model/errors.go:4:2: could not import errors (/usr/local/go/src/errors/wrap.go:8:2: could not import internal/reflectlite (/usr/local/go/src/internal/reflectlite/value.go:8:2: could not import runtime (/usr/local/go/src/runtime/alg.go:8:2: could not import internal/cpu (-: could not load export data: cannot import "internal/cpu" (unknown iexport format version 1), export data is newer version - update tool))))] 
ERRO Running error: buildssa: analysis skipped: errors in package: [/code/pkg/model/defaults.go:4:2: could not import time (/usr/local/go/src/time/format.go:7:8: could not import errors (/usr/local/go/src/errors/wrap.go:8:2: could not import internal/reflectlite (/usr/local/go/src/internal/reflectlite/value.go:8:2: could not import runtime (/usr/local/go/src/runtime/alg.go:8:2: could not import internal/cpu (-: could not load export data: cannot import "internal/cpu" (unknown iexport format version 1), export data is newer version - update tool))))) /code/pkg/model/errors.go:4:2: could not import errors (/usr/local/go/src/errors/wrap.go:8:2: could not import internal/reflectlite (/usr/local/go/src/internal/reflectlite/value.go:8:2: could not import runtime (/usr/local/go/src/runtime/alg.go:8:2: could not import internal/cpu (-: could not load export data: cannot import "internal/cpu" (unknown iexport format version 1), export data is newer version - update tool))))] 
make: *** [Makefile:36: lint] Error 3
```

It works fine with go 1.13! [This issue](https://github.com/kudobuilder/kudo/issues/1380) seems related but I have not really looked into it.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -11,6 +11,8 @@ jobs:
   build:
     name: Build
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
@@ -29,6 +31,8 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    timeout-minutes: 10
+
     steps:
       - uses: actions/setup-go@v1
         with:
@@ -42,6 +46,8 @@ jobs:
   checks:
     name: Checks
     runs-on: ubuntu-latest
+    timeout-minutes: 10
+
     steps:
       - uses: actions/setup-go@v1
         with:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -16,19 +16,12 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
 
     steps:
-      - name: Install Go
-        uses: actions/setup-go@v1
+      - uses: actions/setup-go@v1
         with:
-          go-version: 1.12
-
-      - name: Checkout
-        uses: actions/checkout@v1
-
-      - name: Build
-        run: make build
-
-      - name: Upload
-        uses: actions/upload-artifact@v1
+          go-version: 1.15
+      - uses: actions/checkout@v1
+      - run: make build
+      - uses: actions/upload-artifact@v1
         with:
           name: podsync-${{ matrix.os }}
           path: bin/
@@ -37,30 +30,23 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     steps:
-      - name: Install Go
-        uses: actions/setup-go@v1
+      - uses: actions/setup-go@v1
         with:
-          go-version: 1.12
-      - name: Checkout
-        uses: actions/checkout@v1
-      - name: Test
-        env:
+          go-version: 1.15
+      - uses: actions/checkout@v1
+      - env:
           VIMEO_TEST_API_KEY: ${{ secrets.VIMEO_ACCESS_TOKEN }}
           YOUTUBE_TEST_API_KEY: ${{ secrets.YOUTUBE_API_KEY }}
-        run: |
-          make test
+        run: make test
 
   checks:
     name: Checks
     runs-on: ubuntu-latest
     steps:
-      - name: Install Go
-        uses: actions/setup-go@v1
+      - uses: actions/setup-go@v1
         with:
-          go-version: 1.12
-
-      - name: Checkout
-        uses: actions/checkout@v1
+          go-version: 1.15
+      - uses: actions/checkout@v1
 
       - name: Go mod
         env:
@@ -77,6 +63,4 @@ jobs:
             exit 1
           fi
 
-      - name: Lint
-        run: |
-          make lint
+      - run: make lint

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -11,26 +11,21 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
 
       # Required for the changelog to work correctly
-      - name: Unshallow
-        run: git fetch --prune --unshallow
+      - run: git fetch --prune --unshallow
 
-      - name: Set up Go
-        uses: actions/setup-go@v1
+      - uses: actions/setup-go@v1
         with:
-          go-version: 1.13.x
+          go-version: 1.15
 
-      - name: Docker login
-        env:
+      - env:
           DOCKER_LOGIN: ${{ secrets.DOCKER_LOGIN }}
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
         run: docker login --username "$DOCKER_LOGIN" --password "$DOCKER_PASSWORD"
 
-      - name: Release
-        uses: goreleaser/goreleaser-action@v1
+      - uses: goreleaser/goreleaser-action@v1
         with:
           version: latest
           args: release --rm-dist

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -9,6 +9,7 @@ jobs:
   publish:
     name: Publish
     runs-on: ubuntu-latest
+    timeout-minutes: 10
 
     steps:
       - uses: actions/checkout@v2

--- a/Makefile
+++ b/Makefile
@@ -25,7 +25,7 @@ docker:
 # Pull GolangCI-Lint dependency
 #
 $(GOLANGCI):
-	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(BINPATH) v1.23.1
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(BINPATH) v1.31.0
 	$(GOLANGCI) --version
 
 #

